training/components.py

```diff
@@ -4,7 +4,44 @@
 from kfp import dsl
 from typing import NamedTuple
 from utils.consts import PYTHON_IMAGE
+from typing import Optional
 
+DATA_IMAGE = 'quay.io/shanand/data_processing:0.0.2'
+
+@dsl.component(base_image=DATA_IMAGE)
+def data_processing_op(
```

Comment:
Since `run_data_processing.py` is just one function. Do you think we should just embed it in this component and delete `run_data_processing.py`. This  will match how we do it in `sdg/components.py`. Also you will not need to copy anything into the data processing image or require users to have this very specific image.